salt: Add `bash-completion` for `kubectl` (#3039)
scality/metalk8s: CHANGELOG.md

- Bump etcd version to 3.4.13-0 (PR[#3008](https://github.com/scality/metalk8s/pull/3008))

- [#2996](https://github.com/scality/metalk8s/issues/2996) - The `bash-completion`

>   completions for the `kubectl` command are now setup on every master nodes

Reviewer: ```suggestion
  completions for the `kubectl` command are now provided when `kubectl` is installed
```

>   (PR[#3039](https://github.com/scality/metalk8s/pull/3039))

Reviewer: I noticed this before, shouldn't we leave a space between `PR` and the `#123`?

Reviewer 2: We should.

Author: Hmm don't really know, I just saw this as well some changelog entry have a space other do not have... 
I don't know

Author: Ok I will change this one and will update other ones in another PR

#3040
